src/app/pages/BlockDetailPage/index.tsx

```diff
@@ -53,6 +60,47 @@ export type BlockDetailRuntimeBlock = RuntimeBlock & {
   markAsNew?: boolean
 }
 
+const PrevBlockButton: FC<{ scope: SearchScope; currentRound: number }> = ({ scope, currentRound }) => {
```

Comment:
optional: If we plan to use it in Consensus it's worth to put this into some common components folder.